File: 0137-single-number-ii/0137-single-number-ii.cs
public class Solution {
    public int SingleNumber(int[] nums) {
        for (int i = 0; i < nums.Length; i++) {
            bool isUnique = true;
            for (int j = 0; j < nums.Length; j++) {
                if (i != j && nums[i] == nums[j]) {
                    isUnique = false;
                    break;
                }
            }
            if (isUnique) {
                return nums[i];
            }
        }
        return -1; // fallback in case no unique number is found
    }
}

File: 0137-single-number-ii/0137-single-number-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
    let arr=[]
    for(i=0;i<=nums.length;i++){
        if(nums.indexOf(nums[i])==nums.lastIndexOf(nums[i])){
            return nums[i]
        }
    }
    
};